tools/cldr-code/src/main/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -12,16 +12,7 @@
 
 public final class SubmissionLocales {
     public static Set<String> NEW_CLDR_LOCALES = ImmutableSet.of(
-        "ceb",  // Cebuano (not new in release, but needs major changes)
-        "mai",  // Maithili
-        "mni",  // Manipuri (Bengali script)-Apple as well
-        "sat",  // Santali -(Apple use Olck script)
-        "kok",  // Konkani -(Note: this is already covered by a MS vetter at Modern level)
-        "sd_Deva",   // Sindhi (Devanagari)
-        "su",   // Sundanese (script TBD)
-        "pcm"  // Nigerian Pidgin
-//        "cad",  // Caddo
-//        "gn"    // Guarani
+        // "brx"  // Bodo — ADD this once in Locales.txt
```

Comment:
is NEW_CLDR_LOCALES obsolete? if so, best to remove it for clean code?

Reply:
Probably depends on the release